src/gocept/selenium/webdriver.py

```diff
@@ -89,24 +88,26 @@ def get_firefox_webdriver_args(self):
         if self['headless']:
             options.add_argument('-headless')
 
-        profile = FirefoxProfile(
-            os.environ.get(
-                'GOCEPT_WEBDRIVER_FF_PROFILE',
-                os.environ.get('GOCEPT_SELENIUM_FF_PROFILE')))
-        profile.native_events_enabled = True
-        profile.update_preferences()
```

Comment:
No tests break omitting these two lines.

Reply:
You have still left the line as comments in the file. Maybe add another comment, that this was due to a DeprecationWarning and can be removed after a certain time of no problems.

Reply:
Sorry, this was an oversight, going to remove it.